tests/unit-tests/blocks/test-class-sensei-course-list-filter-block.php

```diff
@@ -40,6 +50,15 @@ class Sensei_Course_List_Filter_Block_Test extends WP_UnitTestCase {
 	 */
 	private $skip_tests = false;
 
+	/**
+	 * Content of the block.
+	 */
+	private $content = '<!-- wp:query {"queryId":13,"query":{"offset":0,"postType":"course","order":"desc","orderBy":"date","author":"","search":"","sticky":"","perPage":4},"displayLayout":{"type":"flex","columns":3},"align":"wide","className":"wp-block-sensei-lms-course-list"} -->
+<div class="wp-block-query alignwide wp-block-sensei-lms-course-list"><!-- wp:sensei-lms/course-list-filter {"type":["categories","featured","student_course"]} /-->
+<!-- wp:post-template {"align":"wide"} -->
+<!-- wp:post-title {"level":1,"isLink":true,"className":"hide-url-underline","fontSize":"large"} /-->
```

Comment:
I removed the `hide-url-underline` class in a previous PR, so I don't think we need it here.

Reply:
Removed it [here](https://github.com/Automattic/sensei/pull/5578/commits/ea03e5aff5ec942a9430f4636a1f9250bcf5fe20), thanks